modules/virt-deploying-operator-cli.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-virt-cli.adoc
+
+[id="virt-deploying-operator-cli_{context}"]
+= Deploying the {VirtProductName} Operator by using the CLI
+
+You can deploy the {VirtProductName} Operator by using the `oc` CLI.
+
+.Prerequisites
+
+* An active subscription to the {VirtProductName} catalog in the `openshift-cnv` namespace.
+
+.Procedure
+
+. Create a file named `hco_cr.yaml` by running the following command:
++
+----
+$ touch hco_cr.yaml
+----
+
+. Open `hco_cr.yaml` in a text editor and paste the following manifest into the file:
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1alpha1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: openshift-cnv
+spec:
+  BareMetalPlatform: true
+----
+
+. Save `hco_cr.yaml` and exit the text editor.
+
+. Deploy the {VirtProductName} Operator by running the following command:
++
+----
+$ oc apply -f hco_cr.yaml
+----
+
+.Verification steps
+
+* Ensure that {VirtProductName} deployed successfully by checking the `PHASE` of the ClusterServiceVersion (CSV). Run the following command:
```

Comment:
Nit: 's/of the ClusterServiceVersion (CSV)/of the ClusterServiceVersion (CSV) in the openshift-cnv namespace'